corehq/apps/hqwebapp/templatetags/hq_shared_tags.py

```diff
@@ -528,3 +529,9 @@ def url_replace(context, field, value):
     params = context['request'].GET.copy()
     params[field] = value
     return params.urlencode()
+
+
+@register.filter
+def view_pdb(element):
+    import ipdb; ipdb.set_trace()
```

Comment:
@esoergel this can be used in view as a filter {{ some_element|view_pdb }} to debug the element. Once finished returns the element.